server/sonar-web/src/main/js/apps/projectActivity/components/forms/AddGraphMetricPopup.tsx

```diff
@@ -33,16 +34,41 @@ interface Props {
 }
 
 export default function AddGraphMetricPopup(props: Props) {
+  let footerNode: React.ReactNode = '';
+  let displayFooterNode: boolean = false;
+
+  if (props.selectedElements.length >= 6) {
+    displayFooterNode = true;
+    footerNode = (
+      <span className="alert alert-info spacer-left spacer-right spacer-top">
+        {translate('project_activity.graphs.custom.add_metric_info')}
+      </span>
+    );
+  } else if (props.metricsTypeFilter != null && props.metricsTypeFilter.length > 0) {
+    displayFooterNode = true;
+    footerNode = (
+      <span className="alert alert-info spacer-left spacer-right spacer-top">
+        {translateWithParameters(
+          'project_activity.graphs.custom.type_x_message',
+          props.metricsTypeFilter
+            .map((type: string) => translate('metric.type', type))
+            .sort()
+            .join(', ')
+        )}
+      </span>
+    );
+  }
+
   return (
     <BubblePopup
       customClass="bubble-popup-bottom-right  bubble-popup-menu abs-width-300"
       position={props.popupPosition}>
       <MultiSelect
-        alertMessage={translate('project_activity.graphs.custom.add_metric_info')}
         allowNewElements={false}
         allowSelection={props.selectedElements.length < 6}
-        displayAlertMessage={props.selectedElements.length >= 6}
+        displayFooterNode={displayFooterNode}
```

Comment:
I think you don't need this `displayFooterNode` at all. In the multiselect you know you have to display a footer node if there is one defined.